### PR TITLE
Add deep dive blurbs

### PR DESCRIPTION
**Pull request by iHiD**

### CODE REVIEW

File: exercises/raindrops/metadata.toml [hunk 1]
```diff
@@ -2,3 +2,5 @@ title = "Raindrops"
 blurb = "Convert a number to a string, the content of which depends on the number's factors."
 source = "A variation on FizzBuzz, a famous technical interview question that is intended to weed out potential candidates. That question is itself derived from Fizz Buzz, a popular children's game for teaching division."
 source_url = "https://en.wikipedia.org/wiki/Fizz_buzz"
+
+deep_dive_blurb = "Explore 14 different ways to solve Raindrops including functional pipelines, matrix multiplication, a precalculated table-based solutions using a macro, an Object Oriented approach, some impressive use of both Python and TCL, and an enterprise edition. Kick back and enjoy 45mins of learning with Jeremy and Erik."
```
Comment (BNAndras) [inline exercises/raindrops/metadata.toml:6]: "Enterprise edition" isn't clear here.

Comment (BNAndras) [inline exercises/raindrops/metadata.toml:6]:
```suggestion
deep_dive_blurb = "Explore 14 different ways to solve Raindrops including functional pipelines, matrix multiplication, a precalculated table-based solutions using a macro, an Object Oriented approach, some impressive use of both Python and TCL, and an enterprise edition. Kick back and enjoy 45 minutes of learning with Jeremy and Erik."
```

File: exercises/reverse-string/metadata.toml [hunk 2]
```diff
@@ -3,3 +3,4 @@ blurb = "Reverse a given string."
 source = "Introductory challenge to reverse an input string"
 source_url = "https://medium.freecodecamp.org/how-to-reverse-a-string-in-javascript-in-3-different-ways-75e4763c68cb"
 deep_dive_youtube_id = "biGAfK6OElE"
+deep_dive_blurb = "Explore 14 different ways to reverse a string, exploring a range of topics including Unicode Codepoints, Graphemes, Stack vs Heap allocations, and pointers. Kick back and enjoy 45mins of learning with Jeremy and Erik."
```
Comment (BNAndras) [inline exercises/reverse-string/metadata.toml:6]:
```suggestion
deep_dive_blurb = "Explore 14 different ways to reverse a string, exploring a range of topics including Unicode codepoints, graphemes, stack vs heap allocations, and pointers. Kick back and enjoy 45 minutes of learning with Jeremy and Erik."
```

File: exercises/leap/metadata.toml [hunk 3]
```diff
@@ -3,3 +3,4 @@ blurb = "Determine whether a given year is a leap year."
 source = "CodeRanch Cattle Drive, Assignment 3"
 source_url = "https://coderanch.com/t/718816/Leap"
 deep_dive_youtube_id = "Fj5m16is5hI"
+deep_dive_blurb = "Deep dive into the different algorithms and approaches you can use to solve Leap, as Jeremy and Erik explore interesting Community Solutions across 10 different languages on Exercism."
```
Comment (IsaacG) [inline exercises/leap/metadata.toml:6]:
```suggestion
deep_dive_blurb = "Deep dive into the different algorithms and approaches you can use to solve Leap. Jeremy and Erik explore interesting Community Solutions across 10 different languages on Exercism."
```

Comment (ErikSchierboom) [reply exercises/leap/metadata.toml:6]: The reverse string deep dive is a bit more enticing I feel:

> Explore 14 different ways to reverse a string, exploring a range of topics including Unicode codepoints, graphemes, stack vs heap allocations, and pointers. Kick back and enjoy 45 minutes of learning with Jeremy and Erik."

Maybe we could try something here too? 

Another difference between the two is that this deep dive refers to the exercise, whereas the `reverse-string` one explains what is being done (reversing a string). Maybe we can choose one or the other? I think the reverse-string one, where we explain _what_ is being done is more interesting to read.

Comment (ErikSchierboom) [reply exercises/leap/metadata.toml:6]: I think adding some examples (like in the reverse-string one) could help, like: "using built in functionality, boolean expressions, modulo operations". Something like that

File: exercises/raindrops/metadata.toml (hunk 1 above)
Comment (ErikSchierboom) [reply exercises/raindrops/metadata.toml:6]:
```suggestion
deep_dive_blurb = "Explore 14 different ways to solve Raindrops including functional pipelines, matrix multiplication, a precalculated table-based solutions using a macro, an Object Oriented approach, some impressive use of both Python and Tcl, and an enterprise edition. Kick back and enjoy 45mins of learning with Jeremy and Erik."
```

Comment (iHiD) [reply exercises/raindrops/metadata.toml:6]: > "Enterprise edition" isn't clear here.

It is when you watch the video 😉 

On a serious note, I'm not sure how else to say it though.

**Comment by iHiD** [comment]
Thanks all. I'll follow up with improvements based on the comments you've all provided, but want to get this merged so we can integrate against it :)

**Pull request merged**